src/main/resources/org/jenkinsci/plugins/github_branch_source/Messages.properties

```
@@ -23,6 +23,7 @@ GitHubBuildStatusNotification.CommitStatusSet=GitHub has been notified of this c
 
 ApiRateLimitChecker.ThrottleForNormalize=Normalize API requests
 ApiRateLimitChecker.ThrottleOnOver=Throttle at/near rate limit
+ApiRateLimitChecker.NoThrottle=Ignore rate limit
```

```suggestion
ApiRateLimitChecker.NoThrottle=Never check rate limit (NOT RECOMMENDED)
```

I don't expect you to do it in this PR, but we should create a help file to describe these.